proposals/dao/optimisticTimelock.ts

```diff
@@ -27,20 +28,58 @@ async function run(addresses, oldContracts, contracts, logging = false) {
   await core.grantRole(role, optimisticTimelock.address);
 }
 
-async function teardown(addresses, oldContracts, contracts, logging) {}
+async function teardown(addresses, oldContracts, contracts, logging) {
+
+    const {
+        tribalChiefOptimisticTimelock
+    } = contracts;
+
+    const {
+        tribalChiefOptimisticMultisigAddress
+    } = addresses;
+    
+    await hre.network.provider.request({
+        method: 'hardhat_impersonateAccount',
+        params: [tribalChiefOptimisticMultisigAddress],
+    });
+
+    await tribalChiefOptimisticTimelock.queueTransaction(
+        '0x956F47F50A910163D8BF957Cf5846D573E7f87CA',
+        '0',
+        'transfer(address,uint256)',
+        '0x000000000000000000000000bc9c084a12678ef5b516561df902fdc426d9548300000000000000000000000000000000000000000000d3c21bcecceda1000000',
+        '1632873600',
+        {from: tribalChiefOptimisticMultisigAddress}
+    );
+
+    await time.increase('1000000');
```

Comment:
are you sure this works? in my dive into the unit tests, I had to convert to a number before passing into time.increase

Reply:
Yeah the execution works